fv3core/.jenkins/install_dace.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -e -x
+orchestration=$1
+if [ -d "gt4py" ]; then rm -rf gt4py; fi
+git clone https://github.com/ai2cm/gt4py.git
+cd gt4py
+git checkout 47021270f067aff314a0344637d090657f1460d8
+cd ../
+pip install -e ./gt4py
+
+if [ -d "dace" ]; then rm -rf dace; fi
+git clone https://github.com/spcl/dace.git
+cd dace
+git checkout 56c42b21b72b043408b725dd85d6a47ba62f451b
+git submodule update --init
+cd ../
+pip install -e ./dace
+
+set +e -x
+# TODO: this somehow does not work with 9 or 10
+module switch gcc gcc/8.3.0
```

Comment:
Should this change be propagated to the module load commands in buildenv for daint?

Reply:
This is a temporary fix, confirmed with @twicki. This should be resolved after the conference paper, and we would like this to be consistent with everything else in buildev.